### ISSUE TITLE
Why the test img pose loss and img expression loss are zero?

### ISSUE THREAD

**Issue by yuanyuan29** [mysee1989/TCAE#15]
I test the pre-trained model on  another dataset and output the different images. However, I find that the img pose loss and img expression loss are always zero. Are there some problems?

The loss of one epoch:
error 0.5489, 
reconstruction loss 0.4762,
val pose img pose loss(target<->pose img) 0.0000, pose img exp loss(input <->pose img) 0.0000
val exp img pose loss(input <->exp img) 0.0000, exp img exp loss(target<->exp img) 0.0000
val out img pose loss(input <->exp img) 0.0000, out img exp loss(target<->exp img) 0.0000
val exp img recon loss(input <->exp img) 0.0276, pose img recon loss(input <->pose img) 0.0408
val exp flow tv loss 0.0042
epoch:-1, current val total_error: 0.54893630743

the test result:
![individualImage (1)](https://user-images.githubusercontent.com/45285331/113658250-44699200-96d2-11eb-9bdb-0f70c25d68ff.png)
![individualImage](https://user-images.githubusercontent.com/45285331/113658280-4cc1cd00-96d2-11eb-8f51-b5e1bd08fdd1.png)

**from lest to right: source image, target image, output image, expression image, pose image**

In addtion, the learned representations and reconstructed images seem strange sometimes, have you obtained a better result?
